static/_redirects

```diff
@@ -466,6 +466,7 @@
 /docs/admin/authorization/     /docs/reference/access-authn-authz/authorization/ 301
 /docs/admin/high-availability/building/     /docs/setup/production-environment/tools/kubeadm/high-availability/   301
 /code-of-conduct/     /community/code-of-conduct/   301
+/values/     /community/values/   301
```

Comment:
I'd use a 302 redirect for shortened URLs, to indicate that one date they might change.

Reply:
Done 👍